src/client/components/DayPicker/DayPicker.DOCUMENTATION.md

```diff
@@ -32,7 +32,17 @@ See react-day-picker [methods reference](http://react-day-picker.js.org/APIMetho
 <DayPicker
   dayPickerRef={ref => (window.picker = ref)}
   onChange={ day => console.log('day:', day) }
-  locale="de"
+  locale="en"
+  disabledDays={{ after: new Date() }}
```

Comment:
Wrong naming? No such prop in prop types neither in component's code.

Reply:
No.

Changed to `modifiers: { disabled: ... }` Fixed.